spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -52,13 +52,56 @@
     end
   end
 
+  context "an ActiveModel with a validation on an attribute that delegates to a has_many association in another ActiveModel" do
```

Comment:
Line is too long. [127/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.